shared/teams/main/banner.js

```diff
@@ -57,12 +57,28 @@ const Banner = ({onReadMore, onHideBanner}: Props) => (
         Read our announcement
       </Text>
     </Box>
-    <Icon
-      type="iconfont-close"
-      onClick={onHideBanner}
-      style={{position: 'absolute', right: globalMargins.tiny, top: globalMargins.tiny}}
-    />
+    <Box
+      style={{
+        ...closeIconStyle,
```

Comment:
might as well put the position / background into the style instead of this spread all the time

Reply:
also whoops didn't mean to commit the bg color!!!

Reply:
might as well not spread it then :)